applications/ssi-toolkit-verifiable-credentials.md

```diff
@@ -0,0 +1,385 @@
+
+Calculation:
+
+| 0.8 | FTE per month |
+| --- | --- |
+| 172 | Work hours per month |
+| 2 | Months |
+| € 130 / hour | Aligned consulting rate |
+| € **35,776 EUR** | **Total cost** |
+| **$ 36,541 USD** |
+
+
+### Milestone 1 Verifiable Credential issue()
+
+- **Estimated duration:** 1 month
+- **FTE:** 0.8
+- **Costs:** $18,000 USD
+
+| **Number** | **Deliverable** | **Specification** |
+| --- | --- | --- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide both inline documentation of the code in Typescript types and TSDoc definition of the issue() function.The main README.md file will show an example flow. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the README.md, we will describe how to run these tests. |
+| 1. | Environment setup | We will set up a developer environment which includes a running EOSIO node that can be used during unit test execution. |
+| 2. | Decision for Verifiable Credential library | We will evaluate the options available for use of an existing verifiable credentials library that can best support the EOSIO protocol. This may happen during step 3 and 4. Results of this discussion will be published in a Github issue. |
+| 3. | Implementation of W3C Vrifiable Condition | This will require us to create a plug-in to the verifiable credential library so that it can support the W3C verifiable conditions Standard as part of the EOSIO DID - allowing the VC to be signed by any permission inside an EOSIO account. |
+| 4. | Implementation of issue() function | We will implement the issue() function that signs a VC with the chosen permission of the EOSIO account. It can be signed by one key, several keys, or a delegated permission. |
+
+### Milestone 2 Verifiable Credential verify()
+
+- **Estimated Duration:** 1 month
+- **FTE:** 0.8
+- **Costs:** $18,000 USD
+
+| **Number** | **Deliverable** | **Specification** |
+| --- | --- | --- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide both inline documentation of the code in Typescript types and TSDoc definition of the verify() function.The main README.md file will show an example flow. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the README.md, we will describe how to run these tests. |
+| 0d. | npm | We will publish the resulting library to npmjs.com for ease of use by the community. This will show an example flow of how to use it. |
+| 0e. | Article | We will publish an article what the library does, why and how to use it. |
+| 1. | Implementation of issue() function | We will implement the verify() function that verifies that the signature of a VC is valid for the EOSIO account and the chosen permission used to authorize it. |
```

Comment:
Typo in the deliverable that still mentions the `issue` method instead of the `verify` method.

Reply:
fixed